reddit_service_websockets/dispatcher.py

```diff
@@ -22,6 +46,14 @@ def __init__(self, metrics):
     def on_message_received(self, namespace, message):
         consumers = self.consumers.get(namespace, [])
 
+        # Compress the message
+        compressed = make_compressed_frame(message, COMPRESSOR)
```

Comment:
More seriously: a simple optimization is to set a minimum message size before we'll even try compressing. The exact thresholds depend on the data, but could be as low as a couple hundred bytes, or something like 1400 for "eh, it'll fit in a  single internet-MTU packet regardless" depending on the goal. Do you think that'd be worthwhile here to save some CPU in the obviously-wont-help cases?